powerstore/datasource_filesystem_schema.go

```diff
@@ -0,0 +1,168 @@
+/*
+Copyright (c) 2025 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Mozilla Public License Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://mozilla.org/MPL/2.0/
+
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package powerstore
+
+import "github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+
+// FileSystemDatasourceSchema is a function that returns the schema for filesystem
+func FileSystemDatasourceSchema() map[string]schema.Attribute {
+	return map[string]schema.Attribute{
+		"access_policy": schema.StringAttribute{
+			MarkdownDescription: "Access Policy",
```

Comment:
please change the description of all these attributes. You can copy from the swagger UI

Reply:
I've changed the description for all the attributes